src/vtktools.jl

```diff
@@ -164,6 +166,20 @@ function calc_node_coordinates(mesh::Trixi.UnstructuredQuadMesh, n_visnodes)
 end
 
 
+function calc_node_coordinates(mesh::P4estMesh, n_visnodes)
+  # Extract number of spatial dimensions
+  ndims_ = ndims(mesh)
+
+  nodes = range(-1, 1, length=n_visnodes)
+
+  node_coordinates = Array{Float64, ndims_+2}(undef, ndims_,
+                                              ntuple(_ -> n_visnodes, ndims_)...,
+                                              Trixi.ncells(mesh))
+
+  return Trixi.calc_node_coordinates!(node_coordinates, mesh, nodes)
```

Comment:
Is `calc_node_coordinates!` correct here or should it be
```suggestion
  return Trixi.calc_tree_node_coordinates!(node_coordinates, mesh, nodes)
```
?

Reply:
No. These are element node coordinates, not tree node coordinates. That's why I renamed all tree related stuff, that became a bit confusing.

Reply:
Got it, thanks